sdk/sample-soap-endpoint/src/main/resources/META-INF/wsdl/sdk.wsdl

```diff
@@ -0,0 +1,146 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+<definitions xmlns="http://schemas.xmlsoap.org/wsdl/" name="CatalogQueryService" targetNamespace="http://ddf.sdk/soap/hello"
+             xmlns:tns="http://ddf.sdk/soap/hello"
+             xmlns:mime="http://schemas.xmlsoap.org/wsdl/mime/"
+             xmlns:http="http://schemas.xmlsoap.org/wsdl/http/"
+             xmlns:soap="http://schemas.xmlsoap.org/wsdl/soap/"
+             xmlns:wsp="http://www.w3.org/ns/ws-policy"
+             xmlns:wsu="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-utility-1.0.xsd"
+             xmlns:wsam="http://www.w3.org/2007/05/addressing/metadata"
+             xmlns:sp="http://docs.oasis-open.org/ws-sx/ws-securitypolicy/200702"
+             xmlns:t="http://docs.oasis-open.org/ws-sx/ws-trust/200512"
+             xmlns:wsaw="http://www.w3.org/2005/08/addressing"
+             xmlns:wsx="http://schemas.xmlsoap.org/ws/2004/09/mex">
+    <types>
+        <xsd:schema targetNamespace="http://ddf.sdk/soap/hello" xmlns:tns="http://ddf.sdk/soap/hello" xmlns:xsd="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="qualified">
+            <xsd:include schemaLocation="HelloWorld.xsd"/>
+        </xsd:schema>
+    </types>
+    <message name="helloWorld">
+        <part name="parameters" element="tns:helloWorld"/>
+    </message>
+    <message name="helloWorldResponse">
+        <part name="parameters" element="tns:helloWorldResponse"/>
+    </message>
+
+    <portType name="HelloWorldServicePort">
+        <operation name="helloWorldOp">
+            <input message="tns:helloWorld"/>
+            <output message="tns:helloWorldResponse"/>
+        </operation>
+    </portType>
+    <binding name="HelloWorldServiceBinding" type="tns:HelloWorldServicePort">
+        <soap:binding style="document" transport="http://schemas.xmlsoap.org/soap/http"/>
+        <wsp:PolicyReference URI="#TransportSAML2Policy" />
+        <operation name="helloWorldOp">
+            <soap:operation soapAction="helloWorld" />
+            <input>
+                <soap:body use="literal"/>
+            </input>
+            <output>
+                <soap:body use="literal"/>
+            </output>
+        </operation>
+    </binding>
+
+    <wsp:Policy wsu:Id="TransportSAML2Policy"
```

Comment:
Would be nice if this could be included both here and as the default policy for any services deployed to ddf (assuming they include the policy vs. hard-code it). Then the tests would stay in sync with any changes to default policies.